.github/workflows/moban-update.yml

```diff
@@ -1,4 +1,7 @@
-on: [push]
```

Comment:
this file itself, would be updated by moban, hence the proper place update it is: https://github.com/moremoban/pypi-mobans/blob/dev/statics/moban-update.yml

Reply:
ok sure! will do that.
Reason for this being that commits were pushed to forks and pr's

Reply:
Ok i don't completely get it.
moban is used to create those files?
How is moban supposed to know the master branch // master repo name?

Reply:
.moban.yml has the answers.

Reply:
https://moban.readthedocs.io/en/latest/ Has the long answer

Reply:
how it works with pyexcel is: 

a template/source file A.jj2 can be stored in either of the following two:

1. pypi-mobans, generic template source
2. pyexcel-mobans, pyexcel specific source

.moban.yml in pyexcel would have:

targets:
  - my_local_A: A.jj2

And to make it a bit complex is:

pyexcel-mobans had a [mobanfile](https://github.com/pyexcel/pyexcel-mobans/blob/master/mobanfile.yaml), where common sections among pyexcel family are stored.

so each individual project only store their specific targets. 

In moban-update.yml case, it is mentioned in https://github.com/pyexcel/pyexcel-mobans/blob/master/mobanfile.yaml#L20 . so if your update is reviewed and committed in pypi-mobans project, moban command read .moban.yml and bring your update forward to pyexcel project.

here is the idea about all above fluffy admin stuff: https://moban.readthedocs.io/en/latest/#at-scale-continous-templating-for-open-source-projects

Reply:
Ok thanks for explination.

i think it'd take me a long way to look up all the internals so I removed that part of the pr.

Main issue being that tests fail on windows (file conflict) and doctests failing in python > 3.6...

Reply:
and the main intention is that workflows are better integrated and much faster